acquisition/devops/README.md

```diff
@@ -0,0 +1,34 @@
+# Remote access to server logs
+
+```bash
+watch -n1 curl "http://localhost:8080/log/<log>?bytes=3000"
+```
+Where ```<log>``` is one of:
+
+- ```stderr``` - Redirected stderr ( server must be started with ```-r``` options ). Bottle writes it output here.
+- ```stdout``` - Redirected stdout ( server must be started with ```-r``` options ).
+- ```debug``` - The servers debug log.
+
+
+# Deploying a source adapter
+
+If there server is started in dev model source adapter can be deployed remotely
```

Comment:
I think you mean "If the server is started in dev mode a source adapter can be deployed"